app/src/features/surveys/observations/observations-table/import-observations/ImportObservationsButton.tsx

```diff
@@ -37,14 +71,14 @@ export const ImportObservationsButton = (props: IImportObservationsButtonProps)
    * @return {*}
    */
   const handleImportObservations = async (file: File) => {
```

Comment:
nit: try catch?

Reply:
Updated to use a regular try/catch + async/await